lib/datasource/docker/quay.ts

```diff
@@ -0,0 +1,55 @@
+import { logger } from '../../logger';
+import { ExternalHostError } from '../../types/errors/external-host-error';
+import * as packageCache from '../../util/cache/package';
+import { http } from './common';
+
+export async function getTagsQuayRegistry(
+  repository: string
+): Promise<string[] | null> {
```

Comment:
I can't see an explicit `return null` below. Can this return signature be simplified?

Reply:
```suggestion
): Promise<string[]> {
```
Yeah, simplify